resources/dicts/patrols/mountainous/med/newleaf.json

```diff
@@ -3330,7 +3330,7 @@
                 "herbs": ["many_herbs", "tansy", "marigold"]
             },
             {
-                "text": "s_c finds a patch of marigold, harvests its best leaves, picks up some nice tansy too, and pads home, unaware of how much of their success rests on their talent for herb gathering.",
+                "text": "s_c finds a patch of marigold, harvests its best leaves, and picks up some nice tansy too. They pad home after their work is finished, unaware of how much of their success rests on their talent for herb gathering.",
```

Comment:
This is more of a comment than a request to change, because it doesn't look like it is using it before - but why aren't these sections using pronoun tagging?

Reply:
Oh, these aren't pronoun tagged because these patrols are a lot older than the pronoun tagging code - all the various herb patrols were written in january 2023 <3